shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/domain/model/Question.kt

```diff
@@ -0,0 +1,40 @@
+package co.nimblehq.avishek.phong.kmmic.domain.model
+
+enum class QuestionDisplayType {
+    INTRO,
+    STAR,
+    HEART,
+    SMILEY,
+    CHOICE,
+    NPS,
+    TEXTAREA,
+    TEXTFIELD,
+    DROPDOWN,
+    OUTRO,
+    UNKNOWN;
+
+    companion object {
+        fun fromString(value: String): QuestionDisplayType {
+            return try {
+                QuestionDisplayType.valueOf(value)
+            } catch(exc: IllegalArgumentException) {
```

Comment:
Please add space between catch and (
```suggestion
            } catch (exc: IllegalArgumentException) {
```

Reply:
Solved in 38b4965